packages/wallet/package.json

```diff
@@ -10,29 +10,30 @@
     "node": "10.16.3"
   },
   "scripts": {
-    "start": "npm run clearContracts && node scripts/start.js",
-    "build": "run-s clearContracts build:typescript build:webpack",
     "build:ci": "yarn build",
     "build:typescript": "npx tsc -b",
     "build:webpack": "CI=false node scripts/build.js",
+    "build": "run-s clearContracts build:typescript build:webpack",
+    "clearContracts": "rm -rf ./build/contracts",
     "contracts:compile": "npx etherlime compile --buildDirectory=./build/contracts",
+    "contracts:deploy:test": "npx etherlime deploy --compile=false --file=./deployment/deploy-test.js",
     "contracts:deploy": "npx etherlime deploy --compile=false",
-    "contracts:deploy:test": "yarn contracts:deploy && npx etherlime deploy --compile=false --file=./deployment/deploy-test.js",
-    "prepare": "yarn build:typescript",
-    "test": "run-s lint:check 'test:app --all'",
-    "test:ci": "yarn run lint:check && yarn run test:ci:contracts && yarn run test:ci:app",
-    "test:ci:app": "yarn test:app --all --ci --runInBand",
-    "test:ci:contracts": "yarn test:contracts --all --ci --runInBand --bail",
-    "test:app": "npx run-jest -c ./config/jest/jest.config.js",
-    "test:contracts": "yarn contracts:compile && yarn contracts:deploy:test && npx jest -c ./config/jest/jest.contracts.config.js",
+    "deployContracts": "NODE_ENV=development npx deploy-contracts",
     "lint:check": "npx tslint --config tslint.json --project .",
     "lint:write": "npx tslint --config tslint.json --project . --fix",
+    "postinstall": "rm -f node_modules/web3/index.d.ts",
+    "prepare": "yarn build:typescript",
     "prettier:check": "npx prettier --check 'src/**/*.{ts,tsx}'",
     "prettier:write": "npx prettier --write 'src/**/*.{ts,tsx}'",
-    "startGanache": "npx etherlime ganache",
-    "postinstall": "rm -f node_modules/web3/index.d.ts",
+    "start": "npm run clearContracts && node scripts/start.js",
+    "startGanache": "npx etherlime ganache --port=8546 > ganache.log",
```

Comment:
We should probably have a simple script that reads in `GANACHE_PORT` and starts `etherlime ganache` so `startGanache` will work when GANACHE_PORT is not 8546